src/components/CardBadge.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components";
+
+const WrapperBadge = styled.div`
+  display: flex;
+  overflow-x: auto;
+  margin: 10px 10px;
+  flex-direction: column;
+  width: 100px;
+`;
+
+const StyledBadge = styled.span`
+  border-radius: 5px;
+  padding: 3px 10px 3px 10px;
+  font-family: "Roboto", sans-serif;
+  border: 1px solid black;
+  font-size: 20px;
+  color: #000000;
```

Comment:
Sometimes you write color names in hex, sometimes in names.
`#000000` === `black`.
I would try to not mix them too often.